panic: assignment to entry in nil map

```
$ "protoc" "--letmegrpc_out=plugins=grpc:/home/aegon/new/go/src" "--proto_path=/home/aegon/new/go/src:." "/home/aegon/new/go/src/unosoft.hu/ws/aeg/pb/spl3/spl3.proto"
panic: assignment to entry in nil map

goroutine 1 [running]:
github.com/gogo/protobuf/protoc-gen-gogo/generator.(*Generator).AddImport(...)
        /home/tgulacsi/go/pkg/mod/github.com/gogo/protobuf@v1.2.1/protoc-gen-gogo/generator/generator.go:603
github.com/gogo/protobuf/protoc-gen-gogo/generator.(*Generator).RecordTypeUse(0xc0000c6000, 0xc0002e8640, 0x1d)
        /home/tgulacsi/go/pkg/mod/github.com/gogo/protobuf@v1.2.1/protoc-gen-gogo/generator/generator.go:2006 +0x11b
github.com/gogo/letmegrpc/html.(*html).Generate(0xc00068a000, 0xc000376b40)
        /home/tgulacsi/go/pkg/mod/github.com/gogo/letmegrpc@v1.0.1-0.20180829130614-1c3379884077/html/html.go:191 +0x25b6
github.com/gogo/protobuf/protoc-gen-gogo/generator.(*Generator).generatePlugin(0xc0000c6000, 0xc000376b40, 0x8942a0, 0xc00068a000)
        /home/tgulacsi/go/pkg/mod/github.com/gogo/protobuf@v1.2.1/protoc-gen-gogo/generator/helper.go:361 +0xf3
github.com/gogo/protobuf/protoc-gen-gogo/generator.(*Generator).GeneratePlugin(0xc0000c6000, 0x8942a0, 0xc00068a000)
        /home/tgulacsi/go/pkg/mod/github.com/gogo/protobuf@v1.2.1/protoc-gen-gogo/generator/helper.go:338 +0x27f
github.com/gogo/protobuf/vanity/command.GeneratePlugin(0xc0000921e0, 0x8942a0, 0xc00068a000, 0x80eef6, 0xd, 0xc000043f88)
        /home/tgulacsi/go/pkg/mod/github.com/gogo/protobuf@v1.2.1/vanity/command/command.go:90 +0x188
main.main()
        /home/tgulacsi/go/pkg/mod/github.com/gogo/letmegrpc@v1.0.1-0.20180829130614-1c3379884077/protoc-gen-letmegrpc/main.go:35 +0x75
--letmegrpc_out: protoc-gen-letmegrpc: Plugin failed with status code 2.
```

--- a/protoc-gen-gogo/generator/generator.go
+++ b/protoc-gen-gogo/generator/generator.go
@@ -482,6 +482,7 @@ func New() *Generator {
 	g.Request = new(plugin.CodeGeneratorRequest)
 	g.Response = new(plugin.CodeGeneratorResponse)
 	g.writtenImports = make(map[string]bool)
+	g.addedImports = make(map[GoImportPath]bool)
 	return g
 }
 

